### PR TITLE
Parse file size (byte) units in all mixes of case

### PR DESCRIPTION
**Pull request by nespera**
I had a go at a fix for #1678. I am an experienced dev, but not at Rust, so I'd be interested in seeing if my test can be expressed in a less clumsy way.

### CODE REVIEW

File: crates/nu-parser/src/parse.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@ use crate::path::expand_path;
 use crate::signature::SignatureRegistry;
 use log::trace;
 use nu_errors::{ArgumentError, ParseError};
+use nu_protocol::hir::Unit::{Gigabyte, Kilobyte, Megabyte, Petabyte, Terabyte};
```
Comment (thegedge) [inline crates/nu-parser/src/parse.rs:8]: No need for this line 🙂

File: crates/nu-parser/src/parse.rs [hunk 2]
```diff
@@ -1119,3 +1120,95 @@ pub fn classify_block(lite_block: &LiteBlock, registry: &dyn SignatureRegistry)
 pub fn garbage(span: Span) -> SpannedExpression {
     SpannedExpression::new(Expression::Garbage, span)
 }
+
+#[test]
+fn unit_parse_byte_units() -> Result<(), ParseError> {
```
Comment (thegedge) [inline crates/nu-parser/src/parse.rs:1125]: 🙌 thanks for adding tests!

File: crates/nu-parser/src/parse.rs (hunk 1 above)
Comment (nespera) [reply crates/nu-parser/src/parse.rs]: Removed

**Comment by thegedge** [comment]
For some reason a build was cancelled, @nespera . I don't know how to manually restart it, so you may have to amend your latest commit and force push to force it to try again.

**Comment by thegedge** [comment]
Awesome, that did the trick. Thanks, @nespera !

**Pull request merged**